docs/overrides/home.html

```diff
@@ -36,7 +44,7 @@ <h1 class="text-2xl font-bold tracking-tight text-black sm:text-4xl">Fast, Decla
 
 
 <div class="overflow-hidden bg-white py-14">
-  <div class="mx-auto max-w-5xl md:px-6 lg:px-8">
+  <div class="mx-auto max-w-7xl md:px-6 lg:px-8">
     <div class="grid grid-cols-1 gap-x-8 gap-y-16 sm:gap-y-20 lg:grid-cols-2 lg:items-start">
       <div class="px-4">
         <div class="mx-auto max-w-2xl lg:mx-0 lg:max-w-lg">
```

Comment:
The sentence "Prepare your environment when entering the shell that has environment already loaded." right below is confusing IMO.

Reply:
I'd like to do a separate language pass. PRs always welcome though!